cdhweb/pages/models.py

```diff
@@ -70,12 +71,12 @@
 EMBED_HELP = """This should be used for videos from Princeton's Media Central. Copy the "oEmbed URL" from the "Share" menu"""
 
 STANDARD_BLOCKS = [
-    ("rich_text", RichText()),
+    ("paragraph", RichText()),
     ("download_block", DownloadBlock()),
     ("cta_block", CTABlock()),
     ("accordion_block", AccordionBlock()),
     ("video_block", Video()),
-    ("hosted_video_block", HostedVideo()),
+    ("embed", HostedVideo()),
```

Comment:
An aside, for if you still want to update these names, you can use the `label` attribute on a `Block`'s  `Meta` class to set the text for this, so it could show up as "hosted video"  but still be referenced as `embed` for the database format :)